app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -171,6 +172,11 @@
         android:textColor="@color/green_shade"
         android:textSize="20sp"
         android:visibility="invisible" />
+
+      <nl.dionsegijn.konfetti.KonfettiView
+        android:id="@+id/banner_confetti_view"
```

Comment:
Suggest making this more specific to the congratulations message (since we'll also have another confetti view in the next PR specifically corresponding to the end-of-exploration experience).

Reply:
Done